lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -656,6 +663,21 @@ def set_severity(record, entry)
       end
     end
 
+    def set_http_request(record, entry)
+      return unless record.key?('httpRequest')
+
+      entry['httpRequest'] = {}
+
+      HTTP_REQUEST_FIELDS.each do |field|
+        if record['httpRequest'].key?(field)
+          entry['httpRequest'][field] = record['httpRequest'][field]
+          record['httpRequest'].delete(field)
+        end
+      end
+
+      record.delete('httpRequest') if record['httpRequest'].empty?
```

Comment:
I wonder if we should unambiguously delete the httpRequest even if it has "unknown" fields.  I don't have a really strong feeling either way, any opinion as a user of it?